dbms/src/Misc/NamesAndTypes.cpp

```diff
@@ -0,0 +1,42 @@
+#include <Core/NamesAndTypes.h>
```

Comment:
Header in Code and implementation in Misc? Is it legal?

Reply:
Only small part of implementation in Misc (to not use DataTypes/DataTypeFactory.h in Core )